scripts/material.js

```diff
@@ -125,17 +125,18 @@
         $(this).find("input").removeClass("focus");
       })
       .on("change", ".form-control-wrapper.fileinput [type=file]", function() {
+        var $this = $(this);
         var value = "";
-        $.each($(this)[0].files, function(i, file) {
```

Comment:
here the `$(this)[0]` is doing an unnecessary waraping use `this.files` directly